src/transformers/models/bigbird_pegasus/modeling_bigbird_pegasus.py

```diff
@@ -51,17 +51,20 @@
 
 _CHECKPOINT_FOR_DOC = "google/bigbird-pegasus-large-arxiv"
 _CONFIG_FOR_DOC = "BigBirdPegasusConfig"
-_TOKENIZER_FOR_DOC = "PegasusTokenizer"
+_TOKENIZER_FOR_DOC = "PegasusTokenizerFast"
```

Comment:
(just a question) any particular reason to use fast tokenizer here, and not in the other 3 models?